cypress/integration/plugins/dashboards-flow-framework/create_workflow_spec.js

```diff
@@ -23,14 +23,14 @@ describe('Creating Workflows Using Various Methods', () => {
         return cy.registerModel({
           body: {
             ...registerModelBody,
-            connector_id: connectorResponse.connector_id,
+            connector_id: connectorResponse.connector_id || 'test_connector_id',
```

Comment:
Nit: Since we already use a mock model search, do we still need to register these models?

Reply:
Thanks - missed the conditional to still run using the actual models if there are no issues on connector and model creation. Updated.